recipes_source/xeon_run_cpu.rst

```diff
@@ -214,12 +214,12 @@ The generic option settings (knobs) include the following:
      - default value
      - help
    * - ``-h``, ``--help``
```

Comment:
Not sure about changes to this file but the rest lgtm

Reply:
fixed!